irods/connection.py

```diff
@@ -473,7 +508,12 @@ def _login_pam(self):
 
         self.send(auth_req)
         # Getting the new password
-        output_message = self.recv()
+        try:
+            output_message = self.recv()
+        # TODO: if TTL setting is <=0 , investigate removing TTL spec in client call & conceivably avoid this exception
+        except irods.exception.PAM_AUTH_PASSWORD_INVALID_TTL as exc:
+            # TODO (#480): raise <new_exception> from <causing_exception> for more succinct error messages
```

Comment:
TODO

Reply:
If TODOs are already issues, they can still appear in "final" drafts of code, or so I thought. Am I correct in believing that?

Reply:
0 means server default.  Negative values shouldn't be attempted.